src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
```

Comment:
> Is it possible?

Yes of course, but it seems that `initStream`/`initEncoder` is never actually called in the constructor.